### PR TITLE
Add Portuguese translations

### PR DESCRIPTION
**Pull request by grote**
Translated by Mateus Baptista, cacarrara and Caio Volpato.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -192,7 +192,7 @@
     </string>
     <string name="notifications_event_alarm_channel_name">Alarms for events</string>
     <string name="notifications_event_alarm_content_text">Alarm for event</string>
-    <string name="notifications_event_alarm_content_title">@string/app_name</string>
+    <string name="notifications_event_alarm_content_title" translatable="false">@string/app_name</string>
```
Comment (grote) [inline app/src/main/res/values/strings.xml]: The problem with having these translatable is that translators will translate the resource identifiers as well to something non-existent. It actually happened for me.

Comment (johnjohndoe) [reply app/src/main/res/values/strings.xml]: Thank you for the translations. What do you suggest with regards to the resource identifiers?

Comment (grote) [reply app/src/main/res/values/strings.xml]: My simple fix was to mark these references as non-translatable like done here in this MR. You could also just copy/duplicate the strings and allow them to be translated differently. Either way, a resource reference shouldn't be translatable.

Comment (johnjohndoe) [reply app/src/main/res/values/strings.xml]: Thank you. I applied your suggestion in 3d5bcd82e5b13e87ef60b94b2b4514050b4c588e.

**Comment by johnjohndoe** [comment]
To simplify the translation process I moved the _alarm time tiles_ to the `strings.xml` file in 1ce6c4b443a98e0b1c1e69a3c43ff1b66ccd7668.

File: app/src/main/res/values-pt/strings.xml [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    <string name="event_online">Evento online</string>
+    <string name="event_detail_section_title_links">Links</string>
+    <string name="alarm_prompt">Alertas</string>
+    <string name="choose_alarm_time">Escolha o horário do alerta</string>
+    <string name="default_alarm_time_title">Escolha o horário de alerta padrão</string>
+    <string name="default_alarm_time_dialog_title">Horário padrão de alerta</string>
+    <string name="reminders">Alertas</string>
+    <string name="reminder_ringtone">Som do alarme</string>
+    <string name="insistent_alarm">Alerta persistente</string>
+    <string name="usage">Dica:\n\n Mantenha pressionado na palestra para ajustar um alerta.</string>
```
[the remark this answers is not in the document]
Comment (grote) [reply app/src/main/res/values-pt/strings.xml]: Done

**Comment by grote** [comment]
Regarding the missing strings from `arrays.xml`, I don't have the time to poke the translators at the moment. Maybe you just take the current translation as a base and set up a Transifex project for it? I can also transfer ownership of the one I had used for those translations.

**Comment by johnjohndoe** [comment]
@grote I think the current translation state is a big win for the project. Therefore I like to thank you again for contributing it and please make sure to share my best regards with Mateus Baptista, cacarrara and Caio Volpato. The array strings can be translated later. I will consider to setup Transifex but I am not sure when I have time to do so.

**Comment by johnjohndoe** [comment]
Happy new year @grote! I started managing translations for the EventFahrplan at [Crowdin](https://crowdin.com/project/eventfahrplan). I like to invite you, Mateus, cacarrara and Caio to join there if you/they like.

**Pull request merged**